modules/ROOT/pages/bootloader-updates.adoc

```diff
@@ -0,0 +1,42 @@
+ .
+= Updating the bootloader
+
+== bootupd
+
+Updating the bootloader is not currently automatic.  The https://github.com/coreos/bootupd/[bootupd]
+project is included in Fedora CoreOS and may be used for manual updates.
+This is usually only relevant on bare metal scenarios, or virtualized
+hypervisors that support Secure Boot.
```

Comment:
Can we expand a bit on why users might want to update, and how they'll know that that situation applies to them?